packages/vector_graphics/lib/src/_http_web.dart

```diff
@@ -0,0 +1,14 @@
+import 'dart:convert';
+// This lint is currently broken for packages that say they support web.
+// ignore: avoid_web_libraries_in_flutter
+import 'dart:html';
+import 'dart:typed_data';
+
+/// Fetches an HTTP resource from the specified [url] using the specified [headers].
+Future<Uint8List> httpGet(Uri uri, {Map<String, String>? headers}) async {
+  final HttpRequest request = await HttpRequest.request(
+    uri.toString(),
+    requestHeaders: headers,
+  );
+  return Uint8List.fromList(utf8.encode(request.responseText!));
```

Comment:
Instead set the responseType to "arraybuffer", then `request.response` should be either a Uint8List or a List<int>

Reply:
grumble grumble in my day you couldn't use that safely because of IE9 grumble grumble

Reply:
back in my day, we used Perl CGI and we _liked_ it!

Reply:
What is this XMLHttpRequest nonsense? AJAX will never take off!